### PR TITLE
add section for microsoft remote desktop

### PR DESCRIPTION
**Pull request by Enlingus**
not sure about location

### Contributor checklist

- [x] My contribution is **ready to be merged** as is
- [x] I verified the modifications **render properly**
- [x] I used **spell checking**
- [x] I introduced **no new warnings** during build or declared them here
- [x] I **used the styling and structure aids available** in [Sphinx/ResT](http://www.sphinx-doc.org/en/stable/rest.html). (Links use `` `…`_``, enumerations `#.`, lists `*`, code ``` ``…`` ```/`.. code::`, warnings .. `warning::`, etc.)
- [x] I reread the text keeping in mind that the text has to be **comprehended** fully **by the target audience**

### CODE REVIEW

File: devops/internal_servers/edge.rst [hunk 1]
```diff
@@ -0,0 +1,234 @@
+Edge Remote Desktop
+===================
+
+Installation
+------------
+
+#. Microsoft provides dedicated virtual machines for testing edge and ie11. Download it `here <https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/>`_ .
+
+   .. note:
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: .. note:**:**

File: devops/internal_servers/edge.rst [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+     
+      Yes Microsoft doesn't provide a simple .iso file. You can just download a prebuilt image for different VM platforms. That's why we have to convert the image.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: Yes**,**

Comment (pgerber) [inline devops/internal_servers/edge.rst]: just → only

File: devops/internal_servers/edge.rst [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+      Suggested is to download the Virtualbox (.ova) image. It actually doesn't matter what image you download but he following steps fit to a Virtualbox image.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: he →the

File: devops/internal_servers/edge.rst [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+
+ 
+#. After downloading we have to convert into the KVM format (qcow2).
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: convert *…* into

File: devops/internal_servers/edge.rst [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+
+   .. figure:: edge/create_vm4.png
+
+   .. figure:: edge/create_vm5.png
+
+   .. figure:: edge/create_vm6.png
+
+   .. note:: 
+      
+      the VitrIO network device won't work for windows because it doesn't have installed the dirvers. That's why we to install them manually.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: **T**he Vi**rt**IO

Comment (pgerber) [inline devops/internal_servers/edge.rst]: we **…** to

File: devops/internal_servers/edge.rst [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+
+
+#. At this moment you should have successfully create a new VM. It should appear on the left side by the other VM's
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: create**d**

File: devops/internal_servers/edge.rst [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+   For the next step we need to command line again. We have to replace to mock image with the windows edge image.
+
+   .. code::
+
+      ssh tadm@host03b.tocco.ch
+
+      VMID=*the nuber you have choosen for you vm*
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: nu**m**ber

Comment (pgerber) [inline devops/internal_servers/edge.rst]: you**r** **VM**

File: devops/internal_servers/edge.rst [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+      exit
+
+      scp vm-102-disk-1.qcow2 tadm@host03b.tocco.ch:/home/tadm/backup/storage/images/${VMID}
+
+      ssh tadm@host03b.tocco.ch
+
+      sudo qm rescan -vmid ${VMID}
+
+
+#. Download divers for the VitrIO network device.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: Vi**rt**IO

File: devops/internal_servers/edge.rst [hunk 9]
```diff
@@ -0,0 +1,234 @@
...
+
+   The VM is now ready to boot, but we need the drivers for the network device else we won't have any connection to the network.
+   Fedora provides the driver for Windows. It can be found `on the fedora project website <https://fedoraproject.org/wiki/Windows_Virtio_Drivers#Direct_download>`_.
+
+   We have to upload the driver the same way we did it with the image file, see step 3.
+   Before we can start the VM we need to add the drivers to a virtual drive, so that you can access it inside the VM.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: VM**,** we

File: devops/internal_servers/edge.rst [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+
+#. The VM should now be ready to boot. 
+   For that, click on the VM on the left side and then on the start button in the right top corner.
+
+   .. figure:: edge/start_vm.png
+
+   .. hint::
+
+      When the vm starts a Windows logo should appear. After booting you should be logged in automatically. 
+      If not (what might be possible it is microsoft), the default password for Microsoft VM's is **Passw0rd!**.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: possible, it

Comment (pgerber) [inline devops/internal_servers/edge.rst]: ~~might be~~ is

File: devops/internal_servers/edge.rst [hunk 11]
```diff
@@ -0,0 +1,234 @@
...
+
+      * choose **Browse my computer for updated driver software**
+
+      * Select the drive with the driver in it. It will scan the drive an automatically recognize the driver and install it.
+
+      * After installing the driver you probably have to reboot the VM.
+
+
+#. IP configuration
+   When we finally got a connection to the network we need to configure the IP address. In the end edge.tocco.ch should point to the VM.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: Now that we …?

File: devops/internal_servers/edge.rst [hunk 12]
```diff
@@ -0,0 +1,234 @@
...
+
+      * Toggle the checkbox to **On** to enable Microsoft Remote Desktop
+
+      * If you go through the form you will see this sentence:
+        **Use this PC name to connect from your remote device**
+
+   Not sure if it has really an effect but to be save change the PC name to edge.
+
+
+#. Now we got the VM connected to the Network and Microsoft Remote Desktop running.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: **n**etwork

File: devops/internal_servers/edge.rst [hunk 13]
```diff
@@ -0,0 +1,234 @@
...
+   To test that you can use the following commands:
+
+   .. code::
+
+      nmap -sP 10.27.1.*
+
+      ping edge.tocco.ch
+
+
+   The solution for this Problem is simple. Just deactivate all firewall's.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: the firewall

File: devops/internal_servers/edge.rst [hunk 14]
```diff
@@ -0,0 +1,234 @@
...
+
+      * Type **firewall** in to the search field at the **left bottom corner**.
+
+      * **Windows Defender Firewall** will appear as a search result. Click on that
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: **,** click on that**.**

File: devops/internal_servers/edge.rst [hunk 15]
```diff
@@ -0,0 +1,234 @@
...
+
+      * Click on the Tab **Turn Windows Defender Firewall on or off** in the list at left side.
+
+      * Disable the firewall **Public and Private** by clicking on the **Turn off** radio button's.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: ~~'s~~

File: devops/internal_servers/edge.rst [hunk 16]
```diff
@@ -0,0 +1,234 @@
...
+
+      * click on the **OK** button to save the setttings. 
+
+      * You probably have to restart the VM again.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: VM**,** again (for emphasis)

File: devops/internal_servers/edge.rst [hunk 17]
```diff
@@ -0,0 +1,234 @@
...
+   .. code::
+
+      rdesktop edge ->
+
+      ERROR: CredSSP: Initialize failed, do you have correct kerberos tgt initialized ?
+      Failed to connect, CredSSP required by server.
+
+  .. attention:: 
+     
+     It is very likely that thiy error will appear now. to get rid of that message hjust disable CredSSP. To avoid that you have to search 1 hour for the checkbox below is shown how to find it.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: . **T**o

Comment (pgerber) [inline devops/internal_servers/edge.rst]: hjust → just

File: devops/internal_servers/edge.rst [hunk 18]
```diff
@@ -0,0 +1,234 @@
...
+
+
+#. Disable CredSSP
+   
+      * Type **System** in the search field at the **left buttom corner**. Click on **System** as it appears.
+
+      * On the **left list** click on the tab **Remote Settings** 
+
+      * A new windows opens. At the bottom you find an enabled check box it is labeled as follows:
+        **Allow connections onyl from computer running Remote Desktop with Network Level Authentification**
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: onyl →only

File: devops/internal_servers/edge.rst [hunk 19]
```diff
@@ -51,17 +51,17 @@ Installation
...
-#. At this moment you should have successfully create a new VM. It should appear on the left side by the other VM's
+#. At this moment you should have successfully created a new VM. It should appear on the left side by the other VM's
    For the next step we need to command line again. We have to replace to mock image with the windows edge image.
 
    .. code::
 
       ssh tadm@host03b.tocco.ch
 
-      VMID=*the nuber you have choosen for you vm*
+      VMID=*the number you have choosen for you VM*
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: you**r**

File: devops/internal_servers/edge.rst [hunk 20]
```diff
@@ -183,19 +183,19 @@ Installation
       ping edge.tocco.ch
 
 
-   The solution for this Problem is simple. Just deactivate all firewall's.
+   The solution for this Problem is simple. Just deactivate all firewall.
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: ~~all~~ **the** firewall

File: devops/internal_servers/edge.rst [hunk 21]
```diff
@@ -183,19 +183,19 @@ Installation
...
 
       * Type **firewall** in to the search field at the **left bottom corner**.
 
-      * **Windows Defender Firewall** will appear as a search result. Click on that
+      * **Windows Defender Firewall** will appear as a search result. **Click on that**
```
Comment (pgerber) [inline devops/internal_servers/edge.rst]: \*\*Click on that\*\***.** (period missing)

File: devops/internal_servers/edge.rst [hunk 22]
```diff
@@ -0,0 +1,234 @@
...
+
+   .. figure:: edge/create_vm4.png
+
+   .. figure:: edge/create_vm5.png
+
+   .. figure:: edge/create_vm6.png
+
+   .. note:: 
+      
+      The VirtIO network device won't work for windows because it doesn't have installed the dirvers. That's why we have to install them manually.
```
Comment (osswald) [inline devops/internal_servers/edge.rst]: dirvers -> drivers

File: devops/internal_servers/edge.rst [hunk 23]
```diff
@@ -0,0 +1,234 @@
...
+
+
+#. At this moment you should have successfully created a new VM. It should appear on the left side by the other VM's
```
Comment (osswald) [inline devops/internal_servers/edge.rst]: VM's -> VMs

File: devops/internal_servers/edge.rst [hunk 24]
```diff
@@ -0,0 +1,234 @@
...
+   For the next step we need to command line again. We have to replace to mock image with the windows edge image.
```
Comment (osswald) [inline devops/internal_servers/edge.rst]: we need to command -> we need the command

Comment (osswald) [inline devops/internal_servers/edge.rst]: to mock -> the mock

File: devops/internal_servers/edge.rst [hunk 25]
```diff
@@ -0,0 +1,234 @@
...
+      exit
+
+      scp vm-102-disk-1.qcow2 tadm@host03b.tocco.ch:/home/tadm/backup/storage/images/${VMID}
+
+      ssh tadm@host03b.tocco.ch
+
+      sudo qm rescan -vmid ${VMID}
+
+
+#. Download divers for the VirtIO network device.
```
Comment (osswald) [inline devops/internal_servers/edge.rst]: willst du tauchen gehen?

File: devops/internal_servers/edge.rst [hunk 26]
```diff
@@ -0,0 +1,234 @@
...
+   Fedora provides the driver for Windows. It can be found `on the fedora project website <https://fedoraproject.org/wiki/Windows_Virtio_Drivers#Direct_download>`_.
+
+   We have to upload the driver the same way we did it with the image file, see step 3.
+   Before we can start the VM, we need to add the drivers to a virtual drive, so that you can access it inside the VM.
+
+   .. figure:: edge//upload_vitrio_driver.png
+
+
+#. The VM should now be ready to boot. 
+   For that, click on the VM on the left side and then on the start button in the right top corner.
```
Comment (osswald) [inline devops/internal_servers/edge.rst]: top right corner

File: devops/internal_servers/edge.rst [hunk 27]
```diff
@@ -0,0 +1,234 @@
...
+
+   .. figure:: edge/start_vm.png
+
+   .. hint::
+
+      When the vm starts a Windows logo should appear. After booting you should be logged in automatically. 
+      If not (what is possible, it is microsoft), the default password for Microsoft VM's is **Passw0rd!**.
```
Comment (osswald) [inline devops/internal_servers/edge.rst]: Microsoft (capital M)

File: devops/internal_servers/edge.rst [hunk 28]
```diff
@@ -0,0 +1,234 @@
...
+
+      * **Properties**
+
+      * open tab Driver
+
+      * click the button **Update Driver**
+
+      * choose **Browse my computer for updated driver software**
+
+      * Select the drive with the driver in it. It will scan the drive an automatically recognize the driver and install it.
```
Comment (osswald) [inline devops/internal_servers/edge.rst]: drive an -> drive and

File: devops/internal_servers/edge.rst [hunk 29]
```diff
@@ -0,0 +1,234 @@
...
+
+      * Click on the Tab **Turn Windows Defender Firewall on or off** in the list at left side.
+
+      * Disable the firewall **Public and Private** by clicking on the **Turn off** radio button.
+
+      * click on the **OK** button to save the setttings. 
+
+      * You probably have to restart the VM, again.
+
+#. Now we can try open a remote desktop.
```
Comment (osswald) [inline devops/internal_servers/edge.rst]: open -> opening

File: devops/internal_servers/edge.rst [hunk 30]
```diff
@@ -0,0 +1,234 @@
...
+   .. code::
+
+      rdesktop edge ->
+
+      ERROR: CredSSP: Initialize failed, do you have correct kerberos tgt initialized ?
+      Failed to connect, CredSSP required by server.
+
+  .. attention:: 
+     
+     It is very likely that thiy error will appear now. To get rid of that message just disable CredSSP. To avoid that you have to search 1 hour for the checkbox below is shown how to find it.
```
Comment (osswald) [inline devops/internal_servers/edge.rst]: thiy -> this

Comment (osswald) [inline devops/internal_servers/edge.rst]: checkbox below -> checkbox. Below

File: devops/internal_servers/edge.rst [hunk 31]
```diff
@@ -0,0 +1,234 @@
...
+
+
+#. Disable CredSSP
+   
+      * Type **System** in the search field at the **left buttom corner**. Click on **System** as it appears.
+
+      * On the **left list** click on the tab **Remote Settings** 
+
+      * A new windows opens. At the bottom you find an enabled check box it is labeled as follows:
```
Comment (osswald) [inline devops/internal_servers/edge.rst]: windows -> window

**Pull request merged**